package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "jsinstallguard",
+  "version": "0.1.5",
+  "description": "Small security wrapper around the package manager which intercepts any preistall and postinstall scripts.",
+  "main": "index.js",
+  "scripts": {
+    "postinstall": "cp ./{.npmrc,.yarnrc,jsig-allow.json} ../../"
```

Comment:
Probably need to consider what happens if someone already has a `.npmrc`, `.yarnrc`, or `.jsig-allow.json` file.

Reply:
Yup, was thinking the same, gits merge capability would be nice